assignments/objects.js

```diff
@@ -49,16 +86,42 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: "Susan",
+  age: 70,
+  // speak: function() {return `Hello, my name is ${this.name!}`}
```

Comment:
```suggestion
  speak: function() {return `Hello, my name is ${this.name}!`}
```
This was correct, you just had the exclamation point inside the expression. ${this.name!} is undefined, there is no property name! . Everything inside the {} is going to be evaluated as an expression. So take the exclamation out of the {} and it's a part of the string instead.